src/lib/support/BytesToHex.cpp

```diff
@@ -208,7 +208,8 @@ void LogBufferAsHex(const char * label, const ByteSpan & span)
     size_t remaining = span.size();
     if (remaining == 0)
     {
-        ChipLogProgress(Support, "%s>>>", ((label != nullptr) ? label : ""));
+        chip::Logging::Log(chip::Logging::kLogModule_NotSpecified, chip::Logging::kLogCategory_None, "%s>>>",
```

Comment:
Why is this exactly needed? I think we shouldn't compile-in logs if logging is disabled.